presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionProcess.java

```diff
@@ -173,7 +173,7 @@ public void close()
 
     public boolean isAlive()
     {
-        return process.isAlive();
+        return process != null && process.isAlive();
```

Comment:
This fix makes sense, but I wonder if CI was failing before the fix? Did it?

Reply:
@mbasmanova  I didn't see the CI failing on this before, probably it's just flaky.

Reply:
I can write a unit test to rule out this possibility.

Reply:
I can write a unit test to rule out this possibility.